doc/sphinx/language/extensions/match.rst

```diff
@@ -639,7 +639,7 @@ Dependent pattern matching
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The examples given so far do not need an explicit elimination
-predicate because all the |rhs| have the same type and Coq
+predicate because all the |rhs| have the same type and |Coq|
```

Comment:
Is the `|rhs|` correct?

Reply:
It's a short-hand defined in this particular file:

```
.. |rhs| replace:: right hand sides
```

Reply:
Hardy seems worth defining the symbol.  It's only used in 2 places in a single paragraph and it requires mental translation when you're looking at the `rst` file.  And you have to remember that it's "right hand side**s**", not "right hand side".  Just a mild rant, don't change it unless you want to.